src/Microsoft.OData.Core/JsonLight/ODataJsonLightPropertySerializer.cs

```diff
@@ -526,6 +527,20 @@ private void ResolveCollectionValueTypeName(ODataCollectionValue collectionValue
             }
         }
 
+        /// <summary>
+        /// Writes a stream property.
+        /// </summary>
+        /// <param name="streamValue">The stream value to be written</param>
+        /// <param name="isOpenPropertyType">If the property is open.</param>
+        private void WriteStreamProperty(ODataBinaryStreamValue streamValue, bool isOpenPropertyType)

```

Comment:
>isOpenPropertyType [](start = 82, length = 18)

isOpenPropertyType is useless?

Reply:
It won't be once we try to decide whether or not to write the property name.

---
In reply to: [259084247](https://github.com/OData/odata.net/pull/1204#discussion_r259084247) [](ancestors = 259084247)